src/update.lua

```diff
@@ -44,7 +44,11 @@ local function update(pl)
         if lastPluginVersion == nil then
             updateFunctions.none()
         else
-            updateFunctions[lastPluginVersion]()
+            for version, func in pairs(updateFunctions) do
```

Comment:
That's looking better! However, the pairs function often doesn't go in order, and all the previous updates will need to be run in order. Complicated problem!

Reply:
I'll switch it to ipairs, as that goes in order.